setup.py

```diff
@@ -56,7 +56,7 @@ def configuration(parent_package='', top_path=None):
 
 setup(
     name="discretize",
-    version="0.3.3",
+    version="0.4.0",
     install_requires=[
         'numpy>=1.7',
         'scipy>=0.13',
```

Comment:
How about taking the opportunity and crank up the scipy-requirement? Ideally at 1.0? But that might be too high. Anyway, 0.13 is a bit low, and leaving Python 2 behind might be a good point to adjust.

Reply:
good call `>= 1`?

Reply:
It might be a bit pushy. But it was released a year ago, and it was a big release (many changes), so it might make it easier...